packages/editor-ui/src/plugins/codemirror/completions/nativesAutocompleteDocs/luxon.instance.docs.ts

```diff
@@ -271,74 +361,121 @@ export const luxonInstanceDocs: Required<NativeDoc> = {
+				hidden: true,
 				section: 'compare',
 				docURL: 'https://moment.github.io/luxon/api-docs/index.html#datetimediffnow',
 				returnType: 'Duration',
 				args: [
 					{ name: 'unit', type: 'string|string[]' },
-					{ name: 'opts', type: 'object' },
+					{ name: 'opts', type: 'Object' },
 				],
 			},
 		},
 		endOf: {
 			doc: {
 				name: 'endOf',
+				description: i18n.baseText('codeNodeEditor.completer.luxon.instanceMethods.endOf'),
 				section: 'edit',
 				docURL: 'https://moment.github.io/luxon/api-docs/index.html#datetimeendof',
 				returnType: 'DateTime',
-				args: [{ name: 'unit', type: 'string', default: "'month'" }],
+				args: [
+					{
+						name: 'unit',
+						description: i18n.baseText(
+							'codeNodeEditor.completer.luxon.instanceMethods.endOf.args.unit',
+						),
+						type: 'string',
+					},
+					{
+						name: 'opts',
+						optional: true,
+						description: i18n.baseText(
+							'codeNodeEditor.completer.luxon.instanceMethods.endOf.args.opts',
+						),
+						default: '{}',
+						type: 'Object',
+					},
+				],
+				examples: [
+					{
+						example: "dt = '2024-03-30T18:49'.toDateTime()\ndt.endOf('month')",
+						evaluated: '2024-03-31T23:59',
+					},
+				],
 			},
 		},
 		equals: {
 			doc: {
 				name: 'equals',
+				description: i18n.baseText('codeNodeEditor.completer.luxon.instanceMethods.equals'),
 				section: 'compare',
 				docURL: 'https://moment.github.io/luxon/api-docs/index.html#datetimeequals',
 				returnType: 'boolean',
-				args: [{ name: 'other', type: 'DateTime' }],
+				args: [
+					{
+						name: 'other',
+						description: i18n.baseText(
+							'codeNodeEditor.completer.luxon.instanceMethods.equals.args.other',
+						),
+						type: 'DateTime',
+					},
+				],
+				examples: [
+					{
+						example:
+							"dt = '2024-03-20T18:49+02:00'.toDateTime()\ndt.equals('2024-03-20T19:49+02:00'.toDateTime())",
+						evaluated: 'false',
+					},
+				],
 			},
 		},
 		hasSame: {
 			doc: {
 				name: 'hasSame',
+				description: i18n.baseText('codeNodeEditor.completer.luxon.instanceMethods.hasSame'),
 				section: 'compare',
 				docURL: 'https://moment.github.io/luxon/api-docs/index.html#datetimehassame',
 				returnType: 'boolean',
 				args: [
-					{ name: 'other', type: 'DateTime' },
-					{ name: 'unit', type: 'string' },
+					{
+						name: 'other',
+						description: i18n.baseText(
+							'codeNodeEditor.completer.luxon.instanceMethods.hasSame.args.other',
+						),
+						type: 'DateTime',
+					},
+					{
+						name: 'unit',
+						description: i18n.baseText(
+							'codeNodeEditor.completer.luxon.instanceMethods.hasSame.args.unit',
+						),
+						type: 'string',
+					},
+				],
+				examples: [
+					{
+						example: "'2024-03-20'.toDateTime().hasSame('2024-03-18'.toDateTime(), 'month')",
+						evaluated: 'true',
+					},
+					{
+						example: "'1982-03-20'.toDateTime().hasSame('2024-03-18'.toDateTime(), 'month')",
+						evaluated: 'false',
```

Comment:
I find Luxon's API rather surprising - it's non-obvious that the year has to be the same as well.